src/core/QuickActionCard/testData.ts

```diff
@@ -0,0 +1,28 @@
+const quickActionTestData = [
+  {
+    src: "",
```

Comment:
You can create an issue for this problem in storybook.

FYI: The problem was local images were not showing up in storybook but images with network URL were.

Reply:
issue already created